app/src/pipelines/generate_referrals/pipeline_wrapper.py

```diff
@@ -55,18 +56,38 @@ def setup(self) -> None:
             ),
             name="prompt_builder",
         )
+        pipeline.add_component("load_supports", components.LoadSupports())
         pipeline.connect("prompt_builder", "llm.messages")
-
+        pipeline.connect("load_supports.supports", "prompt_builder.supports")
         self.pipeline = pipeline
 
     # Called for the `generate-referrals/run` endpoint
-    def run_api(self, query: str) -> dict:
-        supports_from_db = format_support_strings()
+    def run_api(self, query: str, prompt_version_id: str = "") -> dict:
```

Comment:
`prompt_version_id` should be `""` if none is provided. This is what I meant:
```suggestion
    def run_api(self, query: str, prompt_version_id: str = "") -> dict:
        # Retrieve the requested prompt_version_id or error if not found
        try:
            prompt_template = haystack_utils.get_phoenix_prompt(
                "generate_referrals", prompt_version_id
            )
            logger.info("Using the prompt_template: %s", prompt_template)
            response = self.pipeline.run(
              {
                "prompt_builder": {
                   "template": prompt_template,
                   "query": query,
                   "supports": supports_from_db.values(),
                   "resource_json": resource_as_json,
                },
              }
            )
           except Exception as e:
             ...
```